app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/rejectedgoodssingle/CheckYourAnswersController.scala

```diff
@@ -66,6 +66,7 @@ class CheckYourAnswersController @Inject() (
               Ok(
                 checkYourAnswersPage(
                   output,
+                  journey.isAllSelectedDutiesAreCMAEligible,
```

Comment:
Not as elegant but does it job to make reimbursement method hidden.
It disallows selecting wrong reimbursement method when no CMA allowed and satisfies:
https://jira.tools.tax.service.gov.uk/browse/CDSR-1408
https://jira.tools.tax.service.gov.uk/browse/CDSR-1478
https://jira.tools.tax.service.gov.uk/browse/CDSR-1492